src/main/java/com/allegient/candidate/stockquote/domain/QuoteList.java

```diff
@@ -13,36 +13,36 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-package com.allegient.candidate.quoteservice.domain;
+package com.allegient.candidate.stockquote.domain;
 
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class QuoteList {
-    
+
     public static final String DISCLAIMER = "This service is for testing purposes only.  The information returned is randomly generated and does not represent true information.";
-    private List<Quote> quotes;
-    
-    private QuoteList() {
-        super();
+    private final Stream<Quote> quotes;
```

Comment:
I don't think this is a good practice.  A stream is a one time use thing.  I tried this in some of my code too, but eventually went back to using Lists internally because I would run into problems with the stream being closed when I didn't expect it to be closed.  So change this to List<Quote>